File: StockMarketAngular/src/app/models/stockprice.ts
import { Time } from '@angular/common';
import { Timestamp } from 'rxjs';

export class Stockprice {
    stockPriceId: number=0;
    companyCode: number=0;
    stockExchangeId: number=0;
    currentPrice: number=0;
    date: Date;
    time: Date;
}

File: StockMarket/StockMarket.ManageStockExchange.Api/Models/Stockprice.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace StockMarket.ManageStockExchange.Api.Models
{
    public partial class Stockprice
    {
        public int StockpriceId { get; set; }
        public int CompanyCode { get; set; }
        public int StockExchangeId { get; set; }
        public double CurrentPrice { get; set; }
        public DateTime Date { get; set; }
        public TimeSpan Time { get; set; }

        public virtual Stockexchange StockExchange { get; set; }
    }
}
